test-project/Assets/PlaymodeTests/Subscriptions/SubscriptionAggregateCouplingTests.cs

```diff
@@ -0,0 +1,201 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.GameObjectCreation;
+using Improbable.Gdk.Subscriptions;
+using NUnit.Framework;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Improbable.Gdk.PlaymodeTests.Subscriptions
+{
+    public class SubscriptionAggregateCouplingTests
+    {
+        private MockConnectionHandler connectionHandler;
+        private WorkerInWorld workerInWorld;
+        private EntityGameObjectLinker linker;
+
+        private SpatialOSReceiveSystem receiveSystem;
+        private RequireLifecycleSystem requireLifecycleSystem;
+
+        private const string WorkerType = "TestWorkerType";
```

Comment:
we have "TestWorkerType" sprinkled throughout our tests - should there be one place where we define this?

Reply:
It actually doesn't matter at all what you put here 😬